src/static/actions/data.js

```diff
@@ -57,3 +85,39 @@ export function dataFetchProtectedData(token) {
             });
     };
 }
+
+export function dataExecuteAsyncTask() {
+    return (dispatch, state) => {
+        dispatch(dataFetchAsyncTaskRequest());
+        return fetch(`${SERVER_URL}/api/v1/base/async_task`, {
+            method: 'post',
+            headers: {
+                'Accept': 'application/json',
+                'Content-Type': 'application/json'
+            }
+        })
+            .then(checkHttpStatus)
+            .then(parseJSON)
+            .then((response) => {
+                dispatch(dataReceiveAsyncTask());
+                alert(  // eslint-disable-line no-alert
+                    'An asynchronous task was executed in the worker container. Look into the docker console for it :)'
+                );
+            })
+            .catch((error) => {
```

Comment:
Actually, this can be further simplified, and need to remove the dispach to /login